scripts/install_third_party_libs.py

```diff
@@ -36,7 +36,10 @@
     ('future', '0.18.2', os.path.join('third_party', 'python_libs')),
     ('six', '1.16.0', os.path.join('third_party', 'python_libs')),
     ('certifi', '2021.10.8', os.path.join(TOOLS_DIR, 'certifi-2021.10.8')),
-    ('typing-extensions', '4.0.1', os.path.join('third_party', 'python_libs'))
+    # This library is used very frequently so we install it to default location.
```

Comment:
I don't think we can do this -- third_party needs to be self-contained, otherwise you run the risk of having something that works locally but not in prod. So we should be handling all our third-party libs in a single coherent way.

I do see the issue you are running into with imports, though, and the need for a central place to put them. Initially python_utils (and earlier, appengine_config.py) was that "central place", but of course, that's no longer true. But how about, say, limiting it to main.py and the script entry-points (the latter of which, I think, are probably mostly covered by common.py)? If we agree on this we should do a follow-up to clean up other sys.path modifications in random files and forbid that happening in future by lint checks.

@vojtechjelinek @YBCS do you have a suggestion here? I think the above route is the way to go conceptually but we should agree on _where_ the correct entry points are.

Reply:
@YBCS Is this still needed? I thought that after introducing the addition to common.py it should work? `typing-extensions` is already added in PREREQUISITES, so it shouldn't be needed here.